pyrmq/consumer.py

```diff
@@ -93,8 +93,9 @@ def __init__(
         self.max_retries = kwargs.get("max_retries", 20)
         self.error_callback = kwargs.get("error_callback")
         self.infinite_retry = kwargs.get("infinite_retry", False)
+        self.auto_create = kwargs.get("auto_create", True)
```

Comment:
No need. Consumer behavior remains the same only publishers will lose their auto declare